packages/authentication/src/definitions.ts

```diff
@@ -41,6 +41,18 @@ declare module '@capacitor/cli' {
 }
 
 export interface FirebaseAuthenticationPlugin {
+  /**
+   * Changes Firebase Authentication to use the provided app name
+   * Will throw an error if the app has not already been initialized
+   * @param options
+   */
+  useFirebaseApp(options: UseFirebaseAppOptions): Promise<void>;
+
+  /**
+   * Returns the name of the current Firebase App used by Firebase Authentication
+   */
+  currentFirebaseApp(): Promise<FirebaseAppName>;
+
```

Comment:
```suggestion
  /**
   * Changes Firebase Authentication to use the provided app name.
   * Will throw an error if the app has not already been initialized.
   *
   * @since 6.2.0
   */
  setApp(options: SetAppOptions): Promise<void>;
  /**
   * Returns the name of the current Firebase App used by Firebase Authentication.
   *
   * @since 6.2.0
   */
  getApp(): Promise<GetAppResult>;
```